product_sequence/models/product_product.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2004 Tiny SPRL
+# Copyright 2016 Sodexis
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    default_code = fields.Char(
+        required=True,
+        default='/',
+        track_visibility='onchange',
+        help="Set to '/' and save if you want a new internal reference "
+             "to be proposed."
+    )
+
+    _sql_constraints = [
```

Comment:
Because add sql contraints????

this functionality is added on module [product_code_unique](https://github.com/OCA/product-attribute/blob/12.0/product_code_unique/models/product.py#L10)

I prefered keep into 2 modules

Reply:
Yes, I agree